src/js/core.js

```diff
@@ -66,76 +69,26 @@ var vjs = function(id, options, ready){
 
   // Element may have a player attr referring to an already created player instance.
   // If not, set up a new player and return the instance.
-  return tag['player'] || new vjs.Player(tag, options, ready);
+  return tag['player'] || new Player(tag, options, ready);
 };
 
 // Extended name, also available externally, window.videojs
-var videojs = window['videojs'] = vjs;
+// var videojs = window['videojs'] = vjs;
 
 // CDN Version. Used to target right flash swf.
-vjs.CDN_VERSION = 'GENERATED_CDN_VSN';
-vjs.ACCESS_PROTOCOL = ('https:' == document.location.protocol ? 'https://' : 'http://');
+videojs.CDN_VERSION = 'GENERATED_CDN_VSN';
+videojs.ACCESS_PROTOCOL = ('https:' == document.location.protocol ? 'https://' : 'http://');
 
 /**
 * Full player version
 * @type {string}
 */
-vjs['VERSION'] = 'GENERATED_FULL_VSN';
-
-/**
- * Global Player instance options, surfaced from vjs.Player.prototype.options_
- * vjs.options = vjs.Player.prototype.options_
- * All options should use string keys so they avoid
- * renaming by closure compiler
- * @type {Object}
- */
-vjs.options = {
-  // Default order of fallback technology
-  'techOrder': ['html5','flash'],
-  // techOrder: ['flash','html5'],
-
-  'html5': {},
-  'flash': {},
-
-  // Default of web browser is 300x150. Should rely on source width/height.
-  'width': 300,
-  'height': 150,
-  // defaultVolume: 0.85,
-  'defaultVolume': 0.00, // The freakin seaguls are driving me crazy!
-
-  // default playback rates
-  'playbackRates': [],
-  // Add playback rate selection by adding rates
-  // 'playbackRates': [0.5, 1, 1.5, 2],
-
-  // default inactivity timeout
-  'inactivityTimeout': 2000,
-
-  // Included control sets
-  'children': {
-    'mediaLoader': {},
-    'posterImage': {},
-    'loadingSpinner': {},
-    'textTrackDisplay': {},
-    'bigPlayButton': {},
-    'controlBar': {},
-    'errorDisplay': {},
-    'textTrackSettings': {}
-  },
-
-  'language': document.getElementsByTagName('html')[0].getAttribute('lang') || navigator.languages && navigator.languages[0] || navigator.userLanguage || navigator.language || 'en',
```

Comment:
I uh...well...

![that's my bad](http://media.giphy.com/media/10XiIn6nxXJLdS/giphy.gif)